src/Kestrel.Core/Internal/Http/Http1MessageBody.cs

```diff
@@ -362,7 +362,7 @@ private class ForChunkedEncoding : Http1MessageBody
             // byte consts don't have a data type annotation so we pre-cast it
             private const byte ByteCR = (byte)'\r';
             // "7FFFFFFF\r\n" is the largest chunk size that could be returned as an int.
-            private const int MaxChunkPrefixBytes = 10;
+            private const long MaxChunkPrefixBytes = 10;
```

Comment:
Long?

Reply:
The reader's consumed is now `long`. I can keep this as `int` if you like.